src/foundry/foundry.js/clientDocuments/user.d.ts

```diff
@@ -10,7 +13,135 @@ declare global {
    *
    * @param data - Initial data provided to construct the User document
    */
-  class User extends ClientDocumentMixin(foundry.documents.BaseUser) {}
+  class User extends ClientDocumentMixin(foundry.documents.BaseUser) {
+    /**
+     * Track whether the user is currently active in the game
+     */
+    active: boolean;
+
+    /**
+     * Track references to the current set of Tokens which are targeted by the User
+     */
+    targets: UserTargets;
+
+    /**
+     * Track the ID of the Scene that is currently being viewed by the User
+     */
+    viewedScene: string | null;
```

Comment:
From the constructor:
```js
    /**
     * Track the ID of the Scene that is currently being viewed by the User
     * @type {string|null}
     */
    this.viewedScene = data.viewedScene || null;
```
It looks like `viewedScene` is a data-property, but it is not configured in the schema of UserData. Should it be added there or not?

Reply:
https://gitlab.com/foundrynet/foundryvtt/-/issues/5358